src/main/java/com/google/idea/perf/Util.kt

```diff
@@ -31,4 +31,8 @@ inline fun <T> Iterable<T>.sumByLong(selector: (T) -> Long): Long {
 private val formatter = NumberFormat.getInstance()
 fun formatNum(num: Long): String = formatter.format(num)
 fun formatNum(num: Long, unit: String): String = "${formatNum(num)} $unit"
+fun formatNum(num: Double): String = formatter.format(num)
+fun formatNum(num: Double, unit: String): String = "${formatNum(num)} $unit"
+
 fun formatNsInMs(ns: Long): String = formatNum(ns / 1_000_000, "ms")
+fun formatNsInMsWithDecimal(ns: Long): String = formatNum(ns / 1_000_000.0, "ms")
```

Comment:
[optional] For time measurements that we expect to be small, I have a slight preference for using a smaller time unit (like microseconds or nanoseconds) rather than resorting to decimal numbers.